awx/api/serializers.py

```diff
@@ -4664,6 +4664,10 @@ def get_related(self, obj):
 
     def get_summary_fields(self, obj):
         summary_fields = super(ScheduleSerializer, self).get_summary_fields(obj)
+
+        if isinstance(obj.unified_job_template, SystemJobTemplate):
```

Comment:
@mabashian asked for this, you see any problem w/ it @AlanCoding ?

Reply:
check if it's null and stuff like that?

Reply:
If it's `null`, wouldn't `obj.unified_job_template` fail the `isinstance` check?

Reply:
https://github.com/ansible/awx/pull/5392/files#diff-a81324c523b41de7296fdd5ff9063d10R4675

Reply:
oh I see, then that should be okay. I was just glancing quickly over the rest of the serializer